README.md

```diff
@@ -1,44 +1,117 @@
-<a href=https://play.google.com/store/apps/details?id=io.parity.signer/> <img src="./docs/src/res/google-play-badge.png" width="100"/></a>
-<a href=https://itunes.apple.com/us/app/parity-signer/id1218174838> <img src="./docs/src/res/app-store-badge.png" width="100"/></a>
-</center>
+<p align="center">
+    <br><br><br>
+    Air-gapped cold storage for crypto keys.
+</p>
 
-# Parity Signer
+<p align="center">
+    <a href="https://github.com/paritytech/parity-signer/releases"><img src="docs/src/res/github-badge.png" width="120"></a> <a href="https://play.google.com/store/apps/details?id=io.parity.signer"><img src="docs/src/res/google-play-badge.png" width="120"></a> <a href="https://itunes.apple.com/us/app/parity-signer/id1218174838"><img src="docs/src/res/app-store-badge.png" width="120"></a><br<br>
+</p>
 
-Parity Signer is a mobile application that allows any smartphone to act as an air-gapped crypto
-wallet. This is also known as "cold storage".
+# Introduction
 
-You can create accounts in Substrate-based networks, sign messages/transactions, and transfer
-funds to and from these accounts without any sort of connectivity enabled on the device.
+Parity Signer is a mobile application that allows any smartphone to act as an air-gapped crypto wallet. This is also known as "cold storage".
 
-You must turn off or even physically remove the smartphone's Wifi, Mobile Network, and Bluetooth
-to ensure that the mobile phone containing these accounts will not be exposed to any online threat.
-Switching to airplane mode suffices in many cases.
+You can create accounts in Substrate-based networks, sign messages/transactions, and transfer funds to and from these accounts without any sort of connectivity enabled on the device.
 
-# Documentation
+You must turn off or even physically remove the smartphone's Wifi, Mobile Network, and Bluetooth to ensure that the mobile phone containing these accounts will not be exposed to any online threat. Switching to airplane mode suffices in many cases.
 
-Please read instructions before **using Signer for the first time** or before **upgrading**
+☝️ **Disabling the mobile phone's networking abilities is a requirement for the app to be used as intended, check our [wiki](https://paritytech.github.io/parity-signer/about/Security-And-Privacy.html) for more details.**
 
-https://paritytech.github.io/parity-signer/index.html
+Any data transfer from or to the app happens using QR code. By doing so, the most sensitive piece of information, the private keys, will never leave the phone. The Parity Signer mobile app can be used to store any Substrate account, this includes Polkadot (DOT) and Kusama (KSM) networks.
 
-==========================================================
+**Available for both iOS and Android.**
 
-# !!! Parity Signer was refactored from the ground up !!!
+![](docs/src/res/screens-for-readme.png)
```

Comment:
<img width="1500" alt="screens-for-readme" src="https://user-images.githubusercontent.com/8782596/168841475-753265a8-d142-4168-b3d1-e0ab5924df9a.png">

could you try these? these are a bi calmer and - bonus - will work for dark and light gh ui

Reply:
Sure, thanks for making it pretty :)